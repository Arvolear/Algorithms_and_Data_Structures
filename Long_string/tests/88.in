717305561
19570