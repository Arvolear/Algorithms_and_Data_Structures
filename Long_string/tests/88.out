717325131
717285991
14037669828770
36653